src/components/App/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Please, lowercase name of the folder and name of the file

Reply:
Redone!